.circleci/config.yml

```diff
@@ -19,12 +19,16 @@ jobs:
           name: Install build requirements
           command: |
             sudo apt-get install pngquant
+            sudo apt-get install python-enchant
             sudo pip install -r requirements.txt
       - run:
           name: Style Guide Testing
           command: |
             paths=$(git diff origin/master... --name-only)
             sudo python style-test.py $paths
+      - run:
+          name: Spell Checker
+          command: sphinx-build -W -b spelling src build/spelling
```

Comment:
This creates a `spelling` folder in `build` and everything in `build` gets automatically uploaded to the S3 and Cloudfront. We probably don't want that `spelling` folder on the website, so in the `exclude_from_upload` section in the `s3_website.yml` file, add `spelling`, so that folder gets excluded.